submissions/solaryasha/html-css-popup/css/styles.css

```diff
@@ -0,0 +1,193 @@
+body {
+    font-size: 15px;
+    font-family: 'Segoe UI', 'Roboto', arial, sans-serif;
+    overflow: hidden;
```

Comment:
Not a good practice using `overflow: hidden` for `body`.

Reply:
Yeah, it was silly. I have already fixed that.